.travis.yml

```diff
@@ -14,9 +14,11 @@ matrix:
         - go get github.com/dedis/Coding || true
 
       script:
+        - set -e
+        - make test_proto
         - make -C conode verify
-        - env GO111MODULE=on make test
```

Comment:
why this change ?

Reply:
To put all tests that take little time at the beginning. So you don't have to wait for the `make test` that takes a long time, before finding out that your PR fails because you forgot to update the proto-files.